libr/core/cmd_open.c

```diff
@@ -1229,6 +1229,7 @@ static int cmd_open(void *data, const char *input) {
 					eprintf ("%d\n", fd);
 				}
 				r_core_bin_load (core, argv0, baddr);
+				free (file);
```

Comment:
file is not a simple heap pointer. but RCoreFile. this fix is hiding the real problem